Automattic/jetpack
# Fix/admin menu default icon

#### Changes proposed in this Pull Request:


Before this PR, we were previously embedding an SVG in the src. But, the [image broke since we didn't base64 encode it](https://github.com/WordPress/WordPress/blob/master/wp-admin/menu-header.php#L125). When we base64 encoded the image, this would then result some JavaScript functionality that would "paint" the SVG to match the color scheme, which caused issues with the background that we set.

So, this PR moves to pulling in the SVG via URL to bypass the painting and then adjusts CSS to match.

<img width="689" alt="image" src="https://user-images.githubusercontent.com/87168/203790200-65fba2c9-979b-4622-9fe4-859b48e44ecd.png">



#### Other information:


#### Jetpack product discussion



NA

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Create a simple site
* Check out this PR on simple with `bin/jetpack-downloader test jetpack fix/admin-menu-default-icon`
* Go to $site.com/wp-admin and verify that site icon in top left displays correctly (not a broken image)
*  Compare to site icon at https://wordpress.com/stats/$site_slug and verify site icon looks identical
* Create Atomic site
* Use Jetpack Beta plugin to checkout this branch
* Go to $site.com/wp-admin and verify the icon looks correct
